Auctionator.lua

```diff
@@ -4174,7 +4170,7 @@ function Atr_CancelAuction_ByIndex(index)
 
   -- cancel the last item in the list and remove it
 
-  local numInMatchList = #gAtr_Owner_Item_Indices;
+  local numInMatchList, i = #gAtr_Owner_Item_Indices;
```

Comment:
i inside of the for loop below isn't stored (so it was leaked globally) so, can store it up here or in another local line but i just put it here.